### PR TITLE
fixed issue : fetching quotes using genre

### PR DESCRIPTION
**Pull request by Shankhanil**
Solution to issue #11

### CODE REVIEW

File: api/controllers/quotesV2.js [hunk 1]
```diff
@@ -137,6 +137,51 @@ exports.searchByQuote = async (req, res, next) => {
...
 }
 
+exports.searchByGenre = async (req, res, next) => {
+    try {
+
+        const { searchQuery } = req.params;
+        let { author, genre, page = 1, limit = 10 } = req.query;
+
+        const currentPage = parseInt(page);
+        const pageLimit = parseFloat(limit);
```
Comment (pprathameshmore) [inline api/controllers/quotesV2.js:147]: It should be `integer`

**Comment by pprathameshmore** [comment]
Update documentation according to features.

File: api/controllers/quotesV2.js (hunk 1 above)
Comment (Shankhanil) [reply api/controllers/quotesV2.js:147]: ahhh my bad.

Comment (Shankhanil) [reply api/controllers/quotesV2.js]: FIXED!

**Comment by Shankhanil** [comment]
I've made the requested changes. @pprathameshmore

**Pull request merged**